pymeasure/instruments/ametek/ametek7270.py

```diff
@@ -41,6 +57,11 @@ class Ametek7270(Instrument):
         200.0e-3, 500.0e-3, 1.0
     ]
 
+    SENSITIVITIES_IMODE = {0: SENSITIVITIES,
+                           1: list(np.array(SENSITIVITIES) * 1e-6),

```

Comment:
Is it worth to import the full numpy package for that list of constants?
Isn't it better to generate that list once and then copy the results into the file?

Reply:
Well I guessed numpy was probably used in pymeasure but you're right, I modified this with list comprehension:

```
1: [sen * 1e-6 for sen in SENSITIVITIES],
```

Reply:
Yes, that is even better!